src/main/java/MyThread.java

```diff
@@ -0,0 +1,19 @@
+import org.apache.log4j.Logger;
+
+public class MyThread extends Thread {
+    private static final Logger logger = Logger.getLogger(MyThread.class);
+    private Counter count;
```

Comment:
```suggestion
    private Counter counter;
```
same for the other thread

Reply:
done